lib/ddtrace/contrib/grape/endpoint.rb

```diff
@@ -71,7 +71,7 @@ def self.endpoint_run(name, start, finish, id, payload)
             span.resource = resource
 
             # set the request span resource if it's a `rack.request` span
-            request_span = payload[:env][:datadog_rack_request_span]
+            request_span = payload[:env]['datadog.rack_request_span']
```

Comment:
can we make this a constant string so that we can reuse it across the entire integration without mistakes? also this was an internal symbol, since it's expected people access to the request span via the Tracer API (and not accessing directly to the `env` object. I guess this is a safe change, though would be great waiting a bit before introducing this in the upstream.

Thoughts @delner ?

Reply:
I think that's a good consideration, @palazzem. Although if I'm understanding correctly, it's not that someone using this library wants to use this specific `env` key (instead of the Trace API), but that all keys in the `env` are supposed to be strings, for safe iterative operation.

@rbone Could we put this key in a constant and freeze it? Probably in `Datadog::Contrib::Rack::TraceMiddleware`?

Reply:
@delner, @palazzem Thanks for the feedback. I've updated it to use a constant as suggested. Let me know if there's anything further this needs. Otherwise I'll leave it to you to decide when it should be merged; and I'm happy if that's not for awhile yet, we've worked around this issue for now so it's not urgent for us.

Reply:
@rbone thank you very much! we'll let you know when this patch is going to land in one of our releases!